modules/cms-kit/src/Volo.CmsKit.Public.HttpApi/Volo/CmsKit/Public/Blogs/BlogPostPublicController.cs

```diff
@@ -31,8 +33,14 @@ public virtual Task<BlogPostPublicDto> GetAsync(string blogSlug, string blogPost
 
     [HttpGet]
     [Route("{blogSlug}")]
-    public virtual Task<PagedResultDto<BlogPostPublicDto>> GetListAsync(string blogSlug, PagedAndSortedResultRequestDto input)
+    public virtual Task<PagedResultDto<BlogPostPublicDto>> GetListAsync(string blogSlug, BlogPostGetListInput input)
     {
         return BlogPostPublicAppService.GetListAsync(blogSlug, input);
     }
-}
+
+    [HttpGet]
+    public virtual Task<List<CmsUserDto>> GetAuthorsHasBlogPosts()
```

Comment:
It's better to define a route 
```suggestion
    [HttpGet]
    [Route("authors")
    public virtual Task<List<CmsUserDto>> GetAuthorsHasBlogPosts()
```

it'll be `api/cms-kit-public/blog-posts/authors`
instead `api/cms-kit-public/blog-posts`